packages/editor-ui/src/views/WorkflowsView.vue

```diff
@@ -243,13 +245,45 @@ const WorkflowsView = defineComponent({
 		sendFiltersTelemetry(source: string) {
 			(this.$refs.layout as IResourcesListLayoutInstance).sendFiltersTelemetry(source);
 		},
+		saveFiltersOnQueryString() {
+			const queryString = new URLSearchParams();
+
+			if (this.filters.search) {
+				queryString.append('search', this.filters.search);
+			}
+
+			if (this.filters.status !== undefined && this.filters.status !== '') {
+				queryString.append('status', this.filters.status);
+			}
+
+			if (this.filters.tags.length) {
+				queryString.append('tags', this.filters.tags.join(','));
+			}
+
+			void this.$router.push({
```

Comment:
would not `$router.replace` be better here? so that if user clicks `back`, it takes them to previous page instead..

Reply:
With push when I click `back`, it takes me the previous page!